common/defaults.go

```diff
@@ -2130,7 +2130,40 @@ func (r *RateLimitAutoTuneConfig) SetDefaults() error {
 }
 
 func (r *RoutingConfig) SetDefaults() error {
-	if r.ScoreMultipliers != nil {
+	if r.ScoreMultipliers == nil || len(r.ScoreMultipliers) == 0 {
+		r.ScoreMultipliers = []*ScoreMultiplierConfig{
+			// For realtime/unfinalized: prioritize block lag (need fresh data)
+			{
+				Network:  "*",
+				Method:   "*",
+				Finality: []DataFinalityState{DataFinalityStateRealtime, DataFinalityStateUnfinalized},
+
+				ErrorRate:       util.Float64Ptr(4.0),
+				RespLatency:     util.Float64Ptr(6.0),
+				TotalRequests:   util.Float64Ptr(1.0),
+				ThrottledRate:   util.Float64Ptr(3.0),
+				BlockHeadLag:    util.Float64Ptr(8.0),
+				FinalizationLag: util.Float64Ptr(2.0),
+				Misbehaviors:    util.Float64Ptr(5.0),
+				Overall:         util.Float64Ptr(1.0),
+			},
+			// For finalized/unknown: prioritize latency (block lag doesn't matter)
```

Comment:
Maybe let's include:
> Even though "unknown" might include requests for tx hashes or block hashes of tip-of-chain range, they'll be retried due to retryEmpty logic if an upstream doesn't have the data. This is not ideal for numeric getBlockByNumber calls.

Reply:
adding it as a comment here or improving the logic based on this?